Strings/Items/Crimson Tanto.md

```diff
@@ -2,15 +2,16 @@
 
 | Language | Description |
 | :------- | :---------- |
+| Text ID | 2301 / 2302 |
 |||
 | English | <h3>**Crimson Tanto**</h3>A Ceremonial Tanto of unknown origin. Costs Spirit Emblems to use.<br><br>Cut into one's own flesh to aquire Bestowal effect on the Sword.<br><br>This dagger is eerily similar to the Red Mortal Blade, thus it temporarily blocks off resurrective powers when used.<br><br>The blade is inscribed with its true name: "Devoted Flesh"<br><br>One needs to be careful when using this item, as abandoning oneself in search of strength is a tragic thing, indeed.<h3> |
 |||
-| Japanese | <h3>**血汐流し**</h3>出処の知れぬ真っ赤な短刀<br>ＨＰと形代を代償に、血を刀に纏わせる<br>この短刀は不気味なほどに、不死斬りと似通う<br>それゆえか、使えば一時的に回生の力は使えなくなる<br>刻まれた銘は「御贄」<br>それがこの短刀の真の名<br>力のために己を捨てる哀れな覚悟<br>それを持たねば、この短刀は扱えぬ<h3> |
+| Japanese | <h3>**血汐流し**</h3>出処の知れぬ深紅の短刀<br>ＨＰと形代を代償に、血を刀に纏わせる<br><br>この短刀は不気味なほどに、不死斬りと似通う<br>それゆえか、使えば一時的に回生の力は使えなくなる<br>力のために己を捨てる哀れな覚悟<br>それを持たねば、この短刀は扱えぬ<br><br>刻まれた銘は「御贄」<br>それがこの短刀の真の名<h3> |
```

Comment:
【From Kiritanpo】
I prefer 「真っ赤な短刀」 over 「深紅の短刀」
the ceremonial tanto says 「真白い短刀」 so 真っ赤 sounds more jaxtaposed
likewise, I think「刻まれた銘は「御贄」<br>それがこの短刀の真の名」should be the last line, because it is also the case for the ceremonial tanto 
also very minor thing, but extra punctuation on the「出所の知れぬ、真っ赤な短刀」 makes it more Fromsoftware text-ish I think